proxy/http2/Http2Stream.cc

```diff
@@ -671,10 +675,12 @@ Http2Stream::send_response_body(bool call_update)
   Http2ClientSession *parent = static_cast<Http2ClientSession *>(this->get_parent());
 
   if (Http2::stream_priority_enabled) {
+    SCOPED_MUTEX_LOCK(lock, parent->connection_state.mutex, this_ethread());
```

Comment:
is this one necessary?

Reply:
This is not necessary, because `Http2ConnectionState::schedule_stream()` acquire mutex by itself. But I add this for consistency with else block (calling `Http2ConnectionState::send_data_frames()`).